docs-en/chapter_computational_complexity/summary.md

```diff
@@ -2,48 +2,48 @@
 
 **space complexity**
 
-- Space complexity acts similarly to time complexity and is used to measure the tendency of an algorithm's space occupation to grow with the amount of data.
-- The memory space associated with the operation of an algorithm can be categorized into input space, staging space, and output space. Normally, the input space is not counted in the space complexity calculation. The transient space can be divided into instruction space, data space, and stack frame space, of which the stack frame space usually affects the space complexity only in the recursion function.
-- We usually focus only on the worst-case space complexity, i.e., the space complexity of the statistical algorithm with the worst-case input data and the worst-case running time complexity.
-- Common space complexities are $O(1)$, $O(\log n)$, $O(n)$, $O(n^2)$ and $O(2^n)$ in descending order.
+- Space complexity serves similarly to time complexity and is used to measure the trend of the space occupied by an algorithm as the data volume increases.
+- The memory space associated with the operation of an algorithm can be categorized into input space, staging space, and output space. Normally, the input space is not considered in the calculation of space complexity. The transient space can be classified into instruction space, data space, and stack frame space, and the stack frame space usually only affects the space complexity for recursion functions.
+- We mainly focus on the worst-case space complexity, i.e., computing the space complexity with the worst input data and runtime.
+- Common space complexities are $O(1)$, $O(\log n)$, $O(n)$, $O(n^2)$ and $O(2^n)$ from better to worse.
 
 ### Q & A
 
 !!! question "Is the space complexity of tail recursion $O(1)$?"
 
-    Theoretically, the space complexity of a tail recursion function can be optimized to $O(1)$ . However, most programming languages (e.g. Java, Python, C++, Go, C#, etc.) do not support automatic optimization of tail recursion, so the space complexity is usually considered to be $O(n)$ .
+    Theoretically, the space complexity of a tail recursion function can be optimized to $O(1)$. However, most programming languages (e.g., Java, Python, C++, Go, C#, etc.) do not support auto-optimization for tail recursion, so the space complexity is usually considered as $O(n)$.
 
 !!! question "What is the difference between the terms function and method?"
 
-    A function can be executed independently and all arguments are passed explicitly. A method is associated with an object, is implicitly passed to the object that calls it, and can manipulate data contained in instances of the class.
+    A *function* can be executed independently, and all arguments are passed explicitly. On the other hand, A *method* is associated with an object and is implicitly passed to the object that calls it, and can operate data contained in instances of the class.
```

Comment:
"On the other hand," : try not to add meanings that are not present in the original text if it does not serve an important purpose
operate -> manipulate